DOCS/man/options.rst

```diff
@@ -1066,6 +1066,11 @@ Program Behavior
     Enable the builtin script that lets you select from lists of items (default:
     yes). By default, its keybindings start with the ``g`` key.
 
+``--load-context-menu=<yes|no>``
+    Enable the builtin script that implements a context menu. Defaults to
+    ``yes`` on platforms where integration with a native context menu is not
+    implemented, and to ``no`` on platform where it is.
```

Comment:
Should it be `auto` instead? I am not sure if some would prefer the OSD menu than GUI menu.

Reply:
It's not the explicit value `auto`. Is there any reason to make the option special?

Reply:
I have no idea what you mean buy GUI menu.

Reply:
https://github.com/mpv-player/mpv/pull/13700

Reply:
Do you mean Windows users cannot use the ASS menu?